core/src/apps/cardano/byron_addresses.py

```
@@ -22,59 +22,67 @@
+
+
 def _decode_raw(address: bytes) -> bytes:
+    from trezor import log
+
     try:
         address_unpacked = cbor.decode(address)
     except ValueError as e:
         if __debug__:
             log.exception(__name__, e)
-        raise wire.ProcessError("Invalid address")
+        raise ProcessError("Invalid address")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)